performance_manager/lib/rt_trip_updates.py

```diff
@@ -253,34 +263,54 @@ def merge_trip_update_events(
                 ),
             )
 
+    process_logger.log_complete()
+
 
 def process_trip_updates(db_manager: DatabaseManager) -> None:
     """
     process trip updates parquet files from metadataLog table
     """
+    process_logger = ProcessLogger("process_tu")
+    process_logger.log_start()
 
     # pull list of objects that need processing from metadata table
     paths_to_load = get_unprocessed_files(
         "RT_TRIP_UPDATES", db_manager.get_session()
     )
 
+    process_logger.add_metadata(paths_to_load=len(paths_to_load))
+
     for folder_data in paths_to_load.values():
         ids = folder_data["ids"]
         paths = folder_data["paths"]
 
+        subprocess_logger = ProcessLogger(
+            "process_tu_dir", file_count=len(paths)
+        )
+        subprocess_logger.log_start()
+
         try:
             new_events = get_tu_dataframe(paths)
-            new_events = unwrap_tu_dataframe(new_events)
+            new_events_unwrapped = unwrap_tu_dataframe(new_events)
```

Comment:
keeping both these dataframe variables in scope at the same time could be an issue for memory usage

Reply:
ah. that makes a lot of sense. reverting.